views/default/input/access.php

```diff
@@ -66,7 +66,7 @@
 }
 
 // don't call get_default_access() unless we need it
-if (!isset($vars['value']) || $vars['value'] == ACCESS_DEFAULT) {
+if (!isset($vars['value']) || is_null($vars['value']) || $vars['value'] == ACCESS_DEFAULT) {
```

Comment:
Hmm. I must be missing something. Will look again.
On Apr 17, 2015 3:14 PM, "Steve Clay" <notifications@github.com> wrote:

> In views/default/input/access.php
> <https://github.com/Elgg/Elgg/pull/8216#discussion_r28592196>:
>
> > @@ -66,7 +66,7 @@
> >  }
> >
> >  // don't call get_default_access() unless we need it
> > -if (!isset($vars['value']) || $vars['value'] == ACCESS_DEFAULT) {
> > +if (!isset($vars['value']) || is_null($vars['value']) || $vars['value'] == ACCESS_DEFAULT) {
>
> Huh, see http://3v4l.org/0ln88
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8216/files#r28592196>.
>